src/WebComponent.js

```diff
@@ -111,6 +116,7 @@ const attributes = {
 const defaultProps = {
   topics: undefined,
   history: undefined,
+  disableTracking: false,
```

Comment:
should be true by default. I'm not sure if  we wants it activate or not for stelen

Reply:
Agree. and I would call it `enableTracking`.